android/app/src/main/kotlin/coop/polypoly/polypod/features/FeatureManifest.kt

```diff
@@ -8,6 +8,7 @@ open class FeatureManifest(
     val version: String?,
     val description: String?,
     val thumbnail: String?,
+    val thumbnailColor: String?,
```

Comment:
... but it's a String here? Makes sense, but still, using the same name is a bit confusing.

Reply:
Well, this is the _manifest_, which contains a URL to an image represented as a string. The actual _feature_ object has an actual bitmap.